### PR TITLE
[v0.32] docs / ongoing docs for version 0.32.0

### PR DESCRIPTION
**Pull request by dennisocana**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**A description of the changes proposed in the pull request**:

This branch contains upcoming changes we'll merge into the docs on the next release.

- Added initial documentation for `order_override` parameter

image
image

- Merged @rxlxrxsx's PR for updating the parameters configurable on the fly

image

- Initial docs for `ticker` command

image

### CODE REVIEW

File: documentation/docs/strategies/advanced-mm/order-override.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+This feature allows users to customize and specify how Hummingbot creates orders in terms of order levels, spread, and size.
+
+
+## How It Works
+
+Since this is a feature designed for advanced users, it is not configurable from the Hummingbot client. Follow the steps below to set up `order_override` parameter:
+
+1. Edit the strategy config file located in the Hummingbot `conf` folder using a text editor.</br>See [Where are my config and log files?](faq/troubleshooting/#where-are-my-config-and-log-files) in the FAQ page for more information.
+1. The input should be in a dictionary format and the key is user-defined.</br>Make sure there is a space between the colon ( : ) and open bracket ( [ ) as shown in the [Sample Configuration](#sample-configuration) then save your changes.
+1. If Hummingbot is running, run `exit` command and restart so the changes will take effect
```
Comment (Nullably) [inline documentation/docs/strategies/advanced-mm/order-override.md:12]: You can also `stop` then `import` the config file again and `start`.

**Comment by Nullably** [comment]
@dennisocana 
Last Own Trade price type addition is not mentioned here yet, still working on it?

File: documentation/docs/strategies/advanced-mm/order-override.md (hunk 1 above)
Comment (dennisocana) [reply documentation/docs/strategies/advanced-mm/order-override.md]: Thanks @Nullably, done making the changes.

image

**Comment by dennisocana** [comment]
@Nullably it's in the existing docs, Vladimir included it in his PR.

image

**Review by vic-en — APPROVED** [review]
LGTM

**Pull request merged**